django_test_suite.sh

```diff
@@ -4,20 +4,40 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-# exit when any command fails
-set -e
+set -x pipefail
+
+sudo apt-get update -y
+sudo apt-get install -y libmemcached-dev
+
+# Disable buffering, so that the logs stream through.
+export PYTHONUNBUFFERED=1
+
+pip3 install .
+
+export DJANGO_TESTS_DIR="django_tests_dir"
```

Comment:
Looks like a lot of this file is copied from `.kokoro/build.sh`. Doesn't this mean that e.g. django is installed twice?

Reply:
Yeah, that's true. Moved installing Django under the if-statement